File: app/src/main/java/com/hfad/avc/ui/list/contact_list/IContactListViewModel.java
package com.hfad.avc.ui.list.contact_list;

import com.hfad.avc.data.model.Contact;

import java.util.List;

import moxy.MvpView;
import moxy.viewstate.strategy.AddToEndSingleStrategy;
import moxy.viewstate.strategy.StateStrategyType;

@StateStrategyType(AddToEndSingleStrategy.class)
public interface IContactListViewModel extends MvpView {

   void setData(List<Contact> contactsList, int typesort);
}

File: app/src/main/java/com/hfad/avc/ui/list/contact_list/IContactListViewModel.kt
package com.hfad.avc.ui.list.contact_list

import com.hfad.avc.data.model.Contact
import moxy.MvpView
import moxy.viewstate.strategy.AddToEndSingleStrategy
import moxy.viewstate.strategy.StateStrategyType

@StateStrategyType(AddToEndSingleStrategy::class)
interface IContactListViewModel : MvpView {
   fun setData(contactsList: List<Contact>, typeSort: Int)
}